intel_owl/settings/cache.py

```diff
@@ -35,6 +35,8 @@ def get_where(self, starts_with: str, version=None) -> Dict[str, Any]:
         return self.get_many([row[0] for row in rows], version=version)
 
 
+DJANGO_REDIS_IGNORE_EXCEPTIONS = True
```

Comment:
reading the [doc](https://github.com/jazzband/django-redis?tab=readme-ov-file#memcached-exceptions-behavior) about this, I am not sure that's a good idea because if redis is down everything goes down. We can't ignore exceptions

Reply:
Copy pasted from @sevdog, we can clearly change it

Reply:
@drosetti after this check I think we can merge the PR

Reply:
@mlodic removed